docu for modal microfrontends (#454)
kyma-project/luigi: docs/luigi-client-api.md

LuigiClient.linkManager().goBack(true);
```

### openAsModal

Opens a view in a modal. It is possible to specify a title and size. If nothing is specified, the title will be the node label or empty if no node label is set. Default size of the modal is set to large (l), which means 80%. It is also possible to specify m (60%) and s (40%) as modal size. Use it optionally in combination with any of the navigation functions.

#### Parameters

> -   `path` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** path to be navigated to

Reviewer: ```suggestion
-   `path` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** navigation path
```

is the navigation path different from the `path to be navigated to`?

> -   `modalSettings` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** opens a view in a modal. Settings to configure the title and size.

Reviewer: ```suggestion
-   `modalSettings` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** opens a view in a modal. Use these settings to configure the modal's title and size.
```

> -   `modalSettings.title` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** modal title (default is node label or empty if no label is defined)

Reviewer: ```suggestion
-   `modalSettings.title` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** modal title. By default, it is the node label. If there is no label, it is left empty.
```
